lib/src/player.rs

```diff
@@ -72,6 +72,17 @@ pub async fn get_all_like(tag: &str) -> Result<Vec<Player>> {
     Ok(matching_players)
 }
 
+pub async fn get_player_from_slug(slug: &str) -> Result<Player> {
```

Comment:
At the bottom of this file, can you add a test for this newly introduced function? I want to make sure stuff in smithe lib is throughly tested.


It will prob. look somewhat like this:
```
    const SCEND_PLAYER_SLUG: &str = "574b3c34";
    const SCEND_PLAYER_ID: i32 = 28679;

    #[tokio::test]
    #[cfg(feature = "skip_db_tests")]
    async fn test_get_player_from_slug() {
        let res = get_player_from_slug(SCEND_PLAYER_SLUG).await;
        assert!(res.is_ok());
        assert_eq!(res.unwrap().player_id, SCEND_PLAYER_ID);
    }
```